spec/support/shared_examples.rb

```diff
@@ -0,0 +1,35 @@
+shared_examples 'a patcher' do
+  let(:patcher) { described_class.new(patch_location) }
+  let(:patch_location) { nil }
+
+  describe '#mutate_schema' do
+    it 'implements mutate schema' do
+      expect(patcher).to respond_to(:mutate_schema)
```

Comment:
(but then, we find this warns us repeatedly about false positives, and we're getting little benefit anyway, so it's toast)